lisa/node.py

```diff
@@ -381,7 +383,11 @@ def _initialize(self, *args: Any, **kwargs: Any) -> None:
             self._log_handler = create_file_handler(
                 self.local_log_path / "node.log", self.log
             )
-
+        if self.is_test_target:
```

Comment:
It doesn't matter it's test target or not to change the `_first_initialize`. If it needs to skip some real logic, just check `is_test_target` there.

Reply:
got it, updated